traitsui/testing/interactor_registry.py

```diff
@@ -0,0 +1,89 @@
+from traitsui.api import Editor
+
+from traitsui.testing.exceptions import (
+    ActionNotSupported, LocationNotSupported,
+)
+
+
+class InteractionRegistry:
+
+    def __init__(self):
+        self.editor_to_action_to_handler = {}
+        self.editor_to_location_solver = {}
```

Comment:
These two are independent, so we might as well have a registry for interaction, and a separate registry for locations.